solr/core/src/java/org/apache/solr/handler/admin/LukeRequestHandler.java

```diff
@@ -298,7 +299,7 @@ private static String getFieldFlags( SchemaField f )
 
       BytesRef bytes = field.binaryValue();
       if (bytes != null) {
-        f.add( "binary", Base64.byteArrayToBase64(bytes.bytes, bytes.offset, bytes.length));
+        f.add( "binary", StandardCharsets.ISO_8859_1.decode(Base64.getEncoder().encode(bytes.wrapToByteBuffer())).toString());
```

Comment:
I don't think this is correct, why are we decoding something that we just encoded?